woocommerce/woocommerce
# Update admin's webpack config

### All Submissions:






### Changes proposed in this Pull Request:



Update admin's webpack config with the following changes:

- Refactor entryPoints code with a `getEntryPoints()` function to improving readability
- Rename variable `suffix` to `outputSuffix`.
- Use [output.library.type](https://webpack.js.org/configuration/output/#outputlibrarytype) instead as they might drop support for `output.libraryTarget` in the future.
- Merge js rules into single rule
- Remove `'gutenberg-components'` alias, we don't use it in our code now. (Originally added in [this commit](https://github.com/woocommerce/woocommerce-admin/commit/94ec4e04382c45455ce9bf14bd344fb23846e20d))


### How to test the changes in this Pull Request:

1. Run `rm -r plugins/woocommerce/assets/client/admin/* && pnpm nx build woocommerce`
2. See if pages are correctly loaded

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

Dev: Update admin's webpack config

### FOR PR REVIEWER ONLY:
